client/state/data-layer/wpcom/reader/data-request/index.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Internal dependencies
+ */
+import { READER_DATA_REQUEST } from 'state/action-types';
+
+export const readerDataHandler = ( { dispatch }, {
+	dataFetch,
```

Comment:
It looks like we're passing in a function through an action. I'm not sure if this was intentional or not, but it's definitely something we want to be aware of and guard against. If I can help with figuring out a way to avoid this please do let me know!